akka-projection-jdbc/src/main/scala/akka/projection/jdbc/internal/Dialect.scala

```diff
@@ -134,7 +134,8 @@ private[projection] case class MySQLDialect(schema: Option[String], tableName: S
 
   def this(tableName: String) = this(None, tableName)
 
-  private val table = schema.map(s => s"$s.$tableName").getOrElse(tableName)
+  private val table = schema.map(s => s"""$s.$tableName""").getOrElse(s"""$tableName""")
```

Comment:
For the MySQL, we can keep the previous code, we don't need to wrap it with triple-quotes.

Reply:
Done.